### PR TITLE
[INJICERT-421] Manually add changes from release-0.9.0-INJI branch

### PR DESCRIPTION
**Pull request by vharsh**
This PR adds changes made in `release-0.9.0` branch to this branch for the Certify 0.9.1 patch release.

### CODE REVIEW

File: helm/artifactory/templates/configmap-share.yaml [hunk 1]
```diff
@@ -17,10 +17,13 @@ data:
   iam_adapter_regproc_ext_url_env: {{ printf "%s%s" (include "artifactory.baseUrl" .) .Values.artifacts.iamAdapterRegprocExt }}
   zip_file_path: {{ .Values.artifacts.hsmClientZip }}
   cache_provider_url_env: {{ printf "%s%s" (include "artifactory.baseUrl" .) .Values.artifacts.cacheProvider }}
-  runtime_dep_url_env: {{ printf "%s%s" (include "artifactory.baseUrl" .) .Values.artifacts.runtimeDep}}
```
Comment (vharsh) [inline helm/artifactory/templates/configmap-share.yaml:20]: this line was duplicated

File: deploy/install.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@ if [ $# -ge 1 ] ; then
 fi
 
 NS=artifactory
-CHART_VERSION=12.0.3
+CHART_VERSION=0.9.0-INJI-develop
```
Comment (vharsh) [inline deploy/install.sh:10]: Will bump this to `0.9.0-INJI-develop`

File: README.md [hunk 3]
```diff
@@ -16,9 +16,9 @@ All these artifacts are released as a part of the Mosip Release with some of the
 ## Deployment in K8 cluster with other MOSIP services:
 ### Pre-requisites
 * Set KUBECONFIG variable to point to existing K8 cluster kubeconfig file:
-  * ```
-    export KUBECONFIG=~/.kube/<my-cluster.config>
-    ```
+    * ```
```
Comment (jainhitesh9998) [inline README.md]: can you remove spaces

**Review by jainhitesh9998 — APPROVED** [review]
lgtm, all recommended changes done

File: artifacts/Dockerfile [hunk 4]
```diff
@@ -138,6 +141,8 @@ COPY /src/deployment/docker/id-authentication/* ${ida_sh_path}/
 
 COPY /src/hsm/client.zip ${hsm_client_path}/
 
+COPY /src/hsm/client-21.zip ${hsm_client_path}/
+
```
Comment (ckm007) [inline artifacts/Dockerfile:145]: COPY statements to copy images is not added.

Comment (vharsh) [reply artifacts/Dockerfile:145]: I didn't understand the above comment, can you please explain the above?

I've taken the changes from https://github.com/mosip/artifactory-ref-impl/blob/release-0.9.0-INJI/artifacts/Dockerfile

**Review by ckm007 — APPROVED** [review]

**Pull request merged**